packages/graphql-transformer-common/src/ResourceConstants.ts

```diff
@@ -29,16 +29,16 @@ export class ResourceConstants {
     OpenSearchStreamingLambdaFunctionLogicalID: 'OpenSearchStreamingLambdaFunction',
     OpenSearchDataSourceLogicalID: 'OpenSearchDataSource',
 
-    // RDS
-    RDSLambdaIAMRoleLogicalID: 'RDSLambdaIAMRole',
-    RDSLambdaLogAccessPolicy: 'RDSLambdaLogAccessPolicy',
-    RDSPatchingLambdaIAMRoleLogicalID: 'RDSPatchingLambdaIAMRole',
-    RDSLambdaLogicalID: 'RDSLambdaLogicalID',
-    RDSPatchingLambdaLogAccessPolicy: 'RDSPatchingLambdaLogAccessPolicy',
-    RDSPatchingLambdaLogicalID: 'RDSPatchingLambdaLogicalID',
-    RDSLambdaDataSourceLogicalID: 'RDSLambdaDataSource',
-    RDSLambdaDataSourceLogicalName: 'RDSLambdaDatabase',
-    RDSPatchingSubscriptionLogicalID: 'RDSPatchingSubscriptionLogicalID',
+    // SQL
+    SQLLambdaIAMRoleLogicalID: 'SQLLambdaIAMRole',
+    SQLLambdaLogAccessPolicy: 'SQLLambdaLogAccessPolicy',
+    SQLPatchingLambdaIAMRoleLogicalID: 'SQLPatchingLambdaIAMRole',
+    SQLLambdaLogicalID: 'SQLLambdaLogicalID',
+    SQLPatchingLambdaLogAccessPolicy: 'SQLPatchingLambdaLogAccessPolicy',
+    SQLPatchingLambdaLogicalID: 'SQLPatchingLambdaLogicalID',
+    SQLLambdaDataSourceLogicalID: 'SQLLambdaDataSource',
+    SQLLambdaDataSourceLogicalName: 'SQLLambdaDatabase',
+    SQLPatchingSubscriptionLogicalID: 'SQLPatchingSubscriptionLogicalID',
```

Comment:
Asking since these cross the boundary into the V1 package namespace. I assume so, but are all of these config values added after we started on gqlv2 rds? Or is it possible these are expected to be inputs to the old v1 relational stuff? https://docs.amplify.aws/cli-legacy/graphql-transformer/relational/

Reply:
I believe I isolated these to only new V2 symbols & values. I explicitly did NOT change the items relating to previously-existing RDS Data Source support.